portfolio/templates/main/entry.html

```diff
@@ -52,138 +52,169 @@
+                <a href="{{ page.project_url }}" target="_blank">
+                Visit the site
+                <i class="fa fa-external-link" aria-hidden="true"></i>
+                </a>
+            {% endif %}
+            {% if page.infosheet %}
+                | <a href="{{ page.infosheet.url }}" target="_blank">
+                Infosheet 
+                <i class="fa fa-download" aria-hidden="true"></i>
+                </a>
+            {% endif %}
+            <div class="share-widget">
+                <span class="field-label">Share:</span>
+                <a rel="nofollow noopener noreferrer" class="share-window" href="http://www.facebook.com/share.php?u={{ page.full_url }}&title={{ page.title }}" aria-label="Share on Facebook"><i class="fa fa-facebook px-1" aria-hidden="true"></i></a>
+                <a rel="nofollow noopener noreferrer" class="share-window" href="https://twitter.com/intent/tweet?text={{ page.title }}&url={{ page.full_url }}&via=ColumbiaCTL" aria-label="Share on Twitter"><i class="fa fa-twitter px-1" aria-hidden="true"></i></a>
+                <a rel="nofollow noopener noreferrer" class="share-window" href="https://www.linkedin.com/shareArticle?mini=true&url={{ page.full_url }}&title={{ page.title }}&summary={{page.search_description}}"><i class="fa fa-google-plus px-1" aria-label="Share on LinkedIn"></i></a>
+            </div>
+        </div>
+    </section>
+
+    {% if page.overview %}
+    <section class="entry-section overview">
+    <p>{{page.overview}}</p>            
+
+    </section>
+    {% endif %}
+
+    <section class="entry-section">
+    {{page.description|richtext}}
+    </section>
+
+    {% comment %}  #### PROMOTIONAL VIDEO ####  {% endcomment %}
+    {% if page.youtube_video_url %}
+        <section class="entry-section entry-section__video">
+            <div class="video-player embed-responsive embed-responsive-16by9 d-print-none">
+                <iframe src='{{page.youtube_video_url}}' 
+                    title='{{page.video_title}}'
+                    allowfullscreen
+                    frameborder="0"
+                    class="embed-responsive-item">
+                </iframe>
+            </div>
+            <p class="video-attribution d-print-none"> Source: Columbia University</p>
+            <div class="d-none d-print-block print-alert">
+                <p>
+                    This segment of the page contains a video, “{{page.video_title}}”.
+                    The link to this video  is:<br />
+                    {{page.video_url}}
+                </p>
+                <p class="video-attribution"> Source: Columbia University</p>
+            </div>
+        </section>
+    {% else %}{% if page.video_url %}
+        <section class="promotional-video mb-5">
+            <dt class="field-label">
+                Video
+            </dt>
+            <dd>
+                <a title="{{page.video_title}}" href="{{page.video_url}}">{{page.video_title}}</a>
```

Comment:
Should this be `target="_blank"`? Also add rel="noopener noreferrer" if you go this route.